docs/eventing/experimental-features.md

```diff
@@ -85,3 +85,36 @@ With this feature you can allow Knative to resolve the full `APIVersion` and fur
 
 !!! note
     At the moment this feature is implemented only for `Subscription.Spec.Subscriber.Ref`.
+
+### `DeliverySpec.Timeout` field
+
+**Flag name**: `delivery-timeout`
+
+**Stage**: Alpha, disabled by default
+
+**Tracking issue**: https://github.com/knative/eventing/issues/5148
+
+When using the `DeliverySpec` to configure event delivery parameters, you can use `timeout` field to specify the timeout for each sent HTTP request. The duration of the `timeout` parameter is specified using the ISO 8601 format
+
+For example, you can define a `Subscription' that retries 3 times, and on each retry the request timeout is going to be 5 seconds:
```

Comment:
```suggestion
The following example shows a Subscription that retries sending a failed event 3 times, and on each retry the request timeout is 5 seconds:
```

Reply:
I wouldn't use "failed event", because it looks like the event is intrinsically failed. I changed with just "sending an event"